puma/manifest.json

```
@@ -0,0 +1,33 @@
+{
+  "display_name": "Puma",
+  "maintainer": "justin.morris@ferocia.com.au",
+  "manifest_version": "1.0.0",
+  "name": "puma",
+  "metric_prefix": "puma.",
+  "metric_to_check": "puma.workers",
+  "creates_events": false,
+  "short_description": "A fast, concurrent web server for Ruby and Rack",
+  "guid": "93264c0f-a4d1-447d-81b6-bee3eb891df3",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Puma Integration",
+  "categories": [
+    "web"
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "puma",
+  "assets": {
+    "configuration": {
+      "spec": "assets/configuration/spec.yaml"
+    },
+    "dashboards": {},
+    "monitors": {},
+    "saved_views": {},
+    "service_checks": "assets/service_checks.json"
```

```suggestion
    "service_checks": "assets/service_checks.json",
    "metrics_metadata": "metadata.csv"
```
Recently added this field to all integration manifest.json